### ISSUE TITLE
Privacy text not shown if terms and checkboxes are disabled

### ISSUE THREAD

**Issue by mintplugins** [easydigitaldownloads/easy-digital-downloads#6598]
### Expected behavior
When I enable showing the "Privacy" text on the checkout, but not the Privacy checkbox, the text shows up on the checkout. I expect this whether the Terms checkbox is enabled or disabled. 

### Actual behavior
When the terms checkbox is enabled, the Privacy text shows without the checkbox 👍 
But when the terms checkbox is disabled, the Privacy text does not show 

### Steps to reproduce the behavior
1) Disable terms under "Misc" in edd settings
2) Enable "Show the Privacy Policy on checkout" but _not_ "Agree to Privacy Policy"
3) The Privacy text does not show on checkout.

**Comment by cklosowski** [comment]
Sorry for the meta spam. Misunderstood this issue.

The question here is do we follow suit of our Terms which if you don't show the checkbox, we don't show the terms toggle. Should the Privacy Policy follow that example? Or should it show the Privacy Policy toggle even if the checkbox doesn't show up?

**Comment by SDavisMedia** [comment]
Normally I would say don't show the privacy policy text if we're not showing its checkbox. But because we've chosen to show the privacy policy text without its checkbox when terms are enabled, then I think it's a reasonable the privacy policy text to show when terms are disabled. And I would make the same argument for terms text (displaying even without its checkbox).

My opinion here is based solely on the fact that we've chosen to show privacy policy text without its checkbox when terms are enabled.

**Comment by amdrew** [comment]
Came across this issue while logging https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7025 and https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7027

I absolutely agree, I think we're all on the same page. If I enable `Show the Privacy Policy on checkout` from the admin settings, it should show the "Show Privacy Policy" text link at checkout **without any additional set up**. I should not need to enable the `Agree to Terms` admin option to get this to display since they are 2 separate options in 2 separate areas of the admin.

These two separate admin options work differently:

The "Terms of Agreement" section of the admin is all about accepting terms. You can't have agreement text without a checkbox option to physically agree to them. Likewise, you can't show an agreement checkbox option without having any text to agree to.

With the privacy policy, you can absolutely have privacy policy text without needing someone to agree to it (like our privacy policies listed on our websites). But you can't show a checkbox to agree to the privacy policy without showing text to agree to. 

So in summary:

1. You should be able to show the "Show Privacy Policy" text link at checkout by simply enabling the `Show the Privacy Policy on checkout` admin option. It should not rely on any other setting to display.

2. You should be able to show the "Show Privacy Policy" text link AND a "Agree to Privacy Policy?" checkbox to agree to the privacy policy.

3. You should be able to show an "Agree to Terms?" checkbox, plus the actual text to agree to, but you shouldn't be able to only show the text. That's the point of a terms of agreement, you need customers to agree to it before purchasing.

**Comment by SDavisMedia** [comment]
@amdrew @mintplugins This should be taken care of in `issue/7027`.

Please follow testing instructions here: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7027#issuecomment-440413806

**Comment by SDavisMedia** [comment]
Closed via https://github.com/easydigitaldownloads/easy-digital-downloads/pull/7045.
